docs/releaseHistory.md

```diff
@@ -4,6 +4,11 @@
 
 All releases in the v0.x.x series are subject to breaking changes from one version to another.  After the release of v1.0.0, this project will be subject to [semantic versioning](http://semver.org/).
 
+## v0.0.27
+"Bug Fixes and Enhancements"

```

Comment:
Can you make this heading consistent with the others (use asterisks to italicize)?

Reply:
Fixed.